README.md

```diff
@@ -48,11 +51,18 @@ The ideal next step is to test out the [Kontent Sample Spring Boot app](/sample-
 
 ## Publishing
 
-> If you want to publish the new version, you need to have an access to [Bintray's Kentico organization](https://bintray.com/kentico).
+> To publish a new version, it iss required to have write permissions for this repository (to be able to create releases) and access to the [Nexus Repository Manager](https://oss.sonatype.org/).
+
+1. Verify that everything in the branch is ready to be published and the [build and tests](https://github.com/Kentico/kontent-java-packages/actions/workflows/gradle.yml) are passing.
+1. Create new GitHub release - **the tag name of the release will be used as a version**
+    * If you define tag name with "-SNAPSHOT" suffix i.e. `4.4.0-SNAPSHOT` artifact will be published to the `https://oss.sonatype.org/content/repositories/snapshots/`, so that you could use it when you want to test out beta version.

```

Comment:
```suggestion
    * If you define tag name with "-SNAPSHOT" suffix i.e. `4.4.0-SNAPSHOT` artifact will be published to the `https://oss.sonatype.org/content/repositories/snapshots/`, so that you could use it when you want to try out the beta version.

```

or just test